workers_test.go

```diff
@@ -1112,3 +1125,26 @@ func TestWorkers_UpdateWorkerRouteWithNoScript(t *testing.T) {
 	_, err := client.UpdateWorkerRoute(context.Background(), "foo", "e7a57d8746e74ae49c25994dadb421b1", route)
 	assert.NoError(t, err)
 }
+
+func TestWorkers_AttachWorkerToDomain(t *testing.T) {
+	setup(UsingAccount("foo"))
```

Comment:
```suggestion
```

Reply:
This is incorrect; without this line, the tests fail to connect to localhost:
```
--- FAIL: TestWorkers_AttachWorkerToDomain (0.00s)
    workers_test.go:1148:
        	Error Trace:	/Users/aaron/Code/cloudflare-go/workers_test.go:1148
        	Error:      	Received unexpected error:
        	            	HTTP request failed: Put "http://127.0.0.1:59180/accounts/01a7362d577a6c3019a474fd6f485823/workers/domains": dial tcp 127.0.0.1:59180: connect: connection refused
        	Test:       	TestWorkers_AttachWorkerToDomain
FAIL
FAIL	github.com/cloudflare/cloudflare-go	1.076s
FAIL
```